projects/netgrif-components/src/lib/panel/task-panel-single/single-task.component.html

```diff
@@ -36,7 +39,7 @@
         </ng-template>
 
         <ng-template #defaultPageFooter>
-            <mat-card class='bootom-card footer'>
+            <mat-card class='bottom-card footer'>
```

Comment:
class name was changed, but no `.scss` file was changed in this PR. Is this ok?

Reply:
Yes, there was a typo in that word if you see the `bootom` is not the right word.